pymc4/distributions/tensorflow/distribution.py

```diff
@@ -0,0 +1,23 @@
+from ..base.distribution import Distribution
+
+
+class BackendDistribution(Distribution):
+    """
+    BackendDistribution for Tensorflow distributions
+    """
+    __doc__ = Distribution.__doc__ + __doc__
+
+    def sample(self, *args, **kwargs):
+        """
+        Returns
+        ---------
+        Tensor
+        """
+        return self._backend_distribution.sample(*args, **kwargs)
+
+    def sample_numpy(self, *args, **kwargs):
+        """Return raw numpy format of sampling"""
+        return self.sample(*args, **kwargs).numpy()
+
+    def log_prob(self, value):
```

Comment:
`log_prob_numpy` is missing. Better add this to the Statistical Distribution class and override here